### DISCUSSION TITLE
Deploy Pocketbase on Railway in one click!

### DISCUSSION THREAD

**Discussion by FarazPatankar** [pocketbase/pocketbase#2784] [Show and tell]
Hey, we recently launched [Volumes](https://railway.app/changelog/2023-06-23-volumes-priority-boarding#volumes-now-in-priority-boarding) on Railway which unlocks the ability to deploy Pocketbase which is something I've wanted to do for a while.

Deploying it is as simple as clicking the button below.

[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/XfUmjI?referralCode=faraz)

A few things to note:
1. Volumes are still in beta so there may be some hiccups here and there but expect them to be fixed ASAP.
2. I simply used the same Dockerfile in the Fly guide. The code can be seen in this [repo](https://github.com/FarazPatankar/pocketbase).
3. I've only been playing with this for a day so let me know if you run into issues/edge-cases with it.

Finally, we offer a [kickback program](https://railway.app/open-source-kickback) on Railway which means for any money we make from users deploying templates, we kick back 25% of it to the template creator so if that interests you, @ganigeorgiev, I am happy to transfer ownership of the template to you if you're willing to maintain and update it!

**Comment by ganigeorgiev** [comment]
Glad to see persistent file storage is being added to Railway! I'm sure a lot of users will find it useful.

---

> ... I am happy to transfer ownership of the template to you if you're willing to maintain and update it!

Thank you, but similarly to the DO 1-click install app, I don't think I'll have the time to maintain the template, so let's keep the ownership as it is for now.

**Comment by lukebouch** [comment]
I'm interesting in taking over ownership if you don't want to maintain it. This looks super interesting!
